app/controllers/JourneyStartController.scala

```diff
@@ -29,12 +30,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class JourneyStartController @Inject()(customsSessionCacheConnector: CustomsSessionCacheConnector,
                                        identify: IdentifierAction,
+                                       checkEmailIsVerified: EmailAction,
                                        getData: DataRetrievalAction,
                                        sessionRepository: SessionRepository,
                                        mcc: MessagesControllerComponents)(implicit executionContext: ExecutionContext) extends FrontendController(mcc) {
 
 
-  def dutyDeferment(linkId: String): Action[AnyContent] = (identify andThen getData).async { implicit request =>
+  def dutyDeferment(linkId: String): Action[AnyContent] = (identify andThen checkEmailIsVerified andThen getData).async { implicit request =>
```

Comment:
Could you please write a test for this change or update the existing tests to include this change? (if you are not able to do this then leave it)